solver/exporter.go

```diff
@@ -125,6 +129,12 @@ func (e *exporter) ExportTo(ctx context.Context, t CacheExporterTarget, opt Cach
 		res, err := cm.backend.Load(key, v.ID)
 		if err != nil {
 			if errors.Is(err, ErrNotFound) {
+				if i < len(records) {
```

Comment:
Isn't this just supposed to be `v = nil`? Looks like you are incrementing `i` twice.

Reply:
Yes we could just set this to nil, and let the top of the loop increment. I'll update the MR